test/test-communities.rb

```diff
@@ -0,0 +1,290 @@
+class CommunitiesTest < Test::Unit::TestCase
+  def setup
+    @dataset = Datasets::Communities.new
+  end
+
+  def record(*args)
+    Datasets::Communities::Record.new(*args)
+  end
+
+  test('#each') do
+    records = @dataset.each.to_a
+    assert_equal([
+                    1994,
+                    {
+                      :state => 8,
+                      :county => "?",
```

Comment:
It seems that `"?"` is missing value.
How about converting missing value to `nil`?

https://github.com/red-data-tools/red-datasets/blob/master/lib/datasets/hepatitis.rb does so.